modules/local/nf-core-modified/minimap2/align/main.nf

```diff
@@ -29,7 +31,9 @@ process MINIMAP2_ALIGN {
         $args \\
         -t $task.cpus \\
         $index \\
-        $reads > "${prefix}".sam
+        $reads > ${prefix}.sam
+    samtools view -@ $task.cpus -bS ${prefix}.sam | samtools sort -@ $task.cpus -o ${prefix}.sorted.bam
+    samtools index ${prefix}.sorted.bam ${prefix}.bam.bai
```

Comment:
just curious, what are you using the bam files for? or is this just a useful output in case someone wants to view the alignment in IGV or something? or is bioinformatics super dumb and one tool will only take sam and one tool will only take bam 😭 ?

Reply:
Yeah I wanted to save the BAM files in case one wants to look at alignments. Although the coverage calculation with the jgisummarizecontigs only takes BAM from what I remember, so they were for both purposes